lib/pbench/server/api/resources/users_api.py

```diff
@@ -11,6 +11,57 @@
 from pbench.server.api.auth import Auth
 
 
+class PasswordRecovery(Resource):
```

Comment:
You're only using one string JSON parameter, so this isn't a big deal, but if you extended our `ApiBase` class with a schema (`Schema(Parameter("email", ParamType.STRING, required=True))`), the validation (that it exists and isn't null) would be done for you. The main advantage is that this would keep messaging of errors consistent across our APIs.